### DISCUSSION TITLE
How to delete the repo from inactive user which I can not reclaim

### DISCUSSION THREAD

**Discussion by sharelord** [community/community#24605] [Apps, API and Webhooks]
<p>I have created a repo with my official email for our project’s internal use at my last org. Now I want to delete the data from that repo but I can not as I don’t have the access to that email account.<br>
Please suggest what needs to do in this case?</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>If anyone can help there it’s <a href="https://support.github.com/request" class="inline-onebox">GitHub Support</a>. I can’t say what they’d need to verify you’re authorized to access that lost account or delete the data, though.</p>

**Comment by jfullstackdev** [comment]
<p>also, to add, that’s I think is whether you’ve done it because it’s part of your job or it is still your copyright</p>
<p>just like here, in my situation, I did not allow my boss to copy my work in Arduino because it’s my copyright, I’m allowing students because they are my students but my projects are actually not the school’s property</p>
<p>but the content I had no choice but to submit in the form of the Course Syllabus, because that is required of my job but still the actual projects are not included in the submission</p>

**Comment by sharelord** [comment]
<p>Hi,</p>
<p>That was part of my job at that time but the organization reach out to me to delete it as it is associated with my ID but I am not able to access it right now. So is there any way for this?</p>
<p>regards,<br>
Vaibhav</p>

**Comment by jfullstackdev** [comment]
<p>yes, the Support Team, the one that you marked as answer, try opening a ticket there</p>
